tendrl/commons/objects/cluster/atoms/import_cluster/__init__.py

```diff
@@ -122,9 +125,9 @@ def run(self):
                 logger.log(
                     "info",
                     NS.publisher_id,
-                    {"message": "Waiting for participant nodes %s to "
-                                "be "
-                     "imported %s" % (node_list, integration_id)},
+                    {"message": "ImportCluster %s waiting for hosts %s "
+                                "to be "
+                     "imported" % (_cluster.short_name, node_list)},
```

Comment:
This line also should be aligned with previous one

Reply:
done